Source/Code/Kepler/Relativity.Kepler.Template/Services/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Relativity.Api" version="10.3.170.1" targetFramework="net462" />
+  <package id="Relativity.Api" version="10.3.191.8" targetFramework="net462" />
```

Comment:
why is this version 10.3?

Reply:
@calimetikcura We want this to be version 10.3 because per Kepler documentation, kepler is only supported past 10.3.191.8. Please refer to this page - https://platform.relativity.com/RelativityOne/index.htm#Kepler_framework/Template_for_Kepler_services.htm